utility/sort/sort_test.go

```diff
@@ -0,0 +1,25 @@
+package sort
+
+import (
+	"github.com/gojektech/proctor/proctord/jobs/metadata"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestSorting(t *testing.T) {
+	procOne := metadata.Metadata{
+		Name: "one"}
```

Comment:
@RajatVaryani in golang, for multiple line structs, it is convention to end the `}` on the next line.

Reply:
if we were to initiailize struct in one line, that would be written thus:
```
procOne := metadata.Metadata{Name: "one"}
```

Reply:
Done